Readme.md

```diff
@@ -101,8 +101,9 @@ cheese: stilton
 
 You can specify a boolean option long name with a leading `no-` to set the option value to false when used.
 Defined alone this also makes the option true by default.
```

Comment:
In rereading this and the edits, I'm not sure we're using consistent vocabulary on this specific line; i.e. it can be unclear whether "define" (or "specify", "add", "set", etc.) means "when defining the program" or "when invoking the program via the CLI."

It could just be that the sentence is slightly awkwardly worded (and sorry to comment on something that isn't a change in this commit).

Reply:
I agree with the awkward and potential for misinterpretation. I have struggled with the wording a couple of times, but at least we do have examples to consult. I'm not planning to have another go in this PR.